Added Boostrap to the HTML files.

7angsan opened the pull request.

kevinNguyen101 committed:
Added admin draft page

7angsan committed:
Added Boostrap to HTML files.

RipCityBoxer2003 committed:
Bootstrap Download

kevinNguyen101 reviewed (approved):
No apparent issues here. Good design too

RipCityBoxer2003 committed:
Functionality to Add, Delete, and Edit Buttons Completed

Add - Directs to Form
Delete - Pop appears asking user to select event that wants to be deleted
Edit - Directs to Form for now just like Add, looking to implement way of filling the text boxes when you select the event you wish to edit.

RipCityBoxer2003 committed:
Merge branch 'admin-addon' into popupFunctionality

kevinNguyen101 committed:
Merge pull request #2 from upcs/popupFunctionality

Popup Functionality

Baird24 committed:
added style to admin pages

n8hopp committed:
Merge branch 'master' into admin-addon
